### ISSUE TITLE
Webpack 4 issue importing .mjs files

### ISSUE THREAD

**Issue by canadaduane** [vanruesc/postprocessing#346]
### Description of the bug

When importing `postprocessing` in webpack 4, I see many errors such as the following:

```
ERROR in ../node_modules/postprocessing/build/postprocessing.mjs 6246:37-56
Can't import the named export 'WebGLRenderTarget' from non EcmaScript module (only default export is available)

ERROR in ../node_modules/postprocessing/build/postprocessing.mjs 4828:22-34
Can't import the named export 'sRGBEncoding' from non EcmaScript module (only default export is available)
```

I can work around this (partially) by introducing the following module rule to `webpack.config.js`:

```
      {
        test: /\.mjs$/,
        include: /node_modules/,
        type: "javascript/auto",
      },
```

However, doing so causes webpack (typescript?) to duplicate the `three` library, as it appears to be unable to resolve the `three` peer dependency as the same library within both my app's code and postprocessing's code. I see a warning in the JS console:

```
WARNING: Multiple instances of Three.js being imported.
```

And indeed, when building my app, it has two instances of three.js, both at version `0.129.0`.

### To Reproduce

If it would be helpful, I can write a small app to reproduce this, but I don't have time right now.

### Expected behavior

I should be able to `import { ... } from "postprocessing"` in typescript.

### Library versions used

 - Three: `0.129.0`
 - Post Processing: `6.24.0`

### Desktop

 - OS: Linux
 - Browser: Chromium, Firefox, etc.
 - Graphics hardware: Intel

**Comment by vanruesc** [comment]
Thanks for the report.

The package type of `three` was changed to `module` in r137 and I changed the extension of the exported `postprocessing` bundle to `.mjs` which has the same effect. Since this broke compatibility with older version of three, it should be considered a bug. I'll look into possible solutions in a bit.

**Comment by vanruesc** [comment]
Fixed in `postprocessing@6.24.1`.
